packages/grid/package.json

```
@@ -0,0 +1,47 @@
+{
+	"name": "@automattic/grid",
+	"version": "1.0.0",
```

In case we opted for `0.1.0`

```suggestion
	"version": "0.1.0",
```